### ISSUE TITLE
mobile support ?

### ISSUE THREAD

**Issue by monolux** [istvan-ujjmeszaros/bootstrap-autohidingnavbar#22]
Just loaded the test website in a S4 and the demo does not work. Reloaded the demo page a couple of times and cleared the cache. Is this for desktop only?

**Comment by antoniotajuelo** [comment]
Nope, it should work in mobile exactly in the same way as desktop.

**Comment by dwillitzer** [comment]
@monolux Which version android is on the S4 and also the browser type/version as well (Ie. kitkat v4.4.4 & "Internet" v2.1.xxx ) 
@antoniotajuelo I found it working on my note 2 & 3 with kitkat 4.4.4 with Chrome 48.0.254

**Comment by istvan-ujjmeszaros** [comment]
This is an issue with some mobile browsers. More info here:
https://www.tjvantoll.com/2012/08/19/onscroll-event-issues-on-mobile-browsers/

**Comment by istvan-ujjmeszaros** [comment]
Actually that may work, I will give it a try:
http://stackoverflow.com/a/9787064/504270

**Comment by monolux** [comment]
Android version 4.4.2 and mobile Chrome version 48.0.2564.95 ; GT-I9505 Build/KOT49H
